x-pack/plugins/apm/public/components/shared/StickyProperties/index.tsx

```diff
@@ -56,6 +56,7 @@ PropertyValue.displayName = 'PropertyValue';
 const PropertyValueTruncated = styled.span`
   display: inline-block;
   line-height: ${px(unit)};
+  padding-bottom: ${px(units.eighth)};
```

Comment:
It looks like the root cause is that we have a line-height identical to the font-size. Typically the line-height needs to be 10-20% larger than the font-size, to avoid any glyphs bleeding outside. 
So instead if adding a padding, I suggest we set the line-height to 120% of the font-size:
```css
line-height: 1.2;
```

Reply:
also, i'm not sure why we need to set `display: inline-block` but the clipping bug goes away when we get rid of that style as well.